src/SubjectFilter.php

```
@@ -377,6 +380,27 @@ protected function failed(Spec $spec): Failure
+            $lastFailure = null;
+            foreach ($spec->filter()->getFailures() as $subField => $failures) {
+                foreach ($failures as $failure) {
+                    $lastFailure = $this->failures->add(
+                        $subField,
+                        $failure->getMessage(),
+                        $failure->getArgs()
+                    );
+                }
+            }
+            if ($lastFailure !== null) {
+                return $lastFailure;
```

PHPStan (2.1.54)

[error] Method Aura\Filter\SubjectFilter::failed() should return Aura\Filter\Failure\Failure but returns Aura\Filter_Interface\FailureInterface.


(return.type)